test/test_notebook.py

```diff
@@ -68,6 +68,14 @@ def test_agentchat_function_call(save=False):
     run_notebook("agentchat_function_call.ipynb", save=save)
 
 
+@pytest.mark.skipif(
```

Comment:
If it's just for async function_call test and isn't meaned for tutorial purpose, maybe move the code into a test case instead of putting it inside a notebook?

Reply:
Good idea. Let me do that.

Reply:
@LittleLittleCloud could I get a review for this. Would like to get this merged -- somebody just asked for an example for this, so nb is useful.